content/en/status/_index.md

```diff
@@ -54,11 +54,11 @@ Checking the current status for each client in the README of its [github repo](h
 **API:** stable  
 **SDK:** stable  
 **Protocol:** stable  
-**Collector:** stable  
+**Collector:** experimental  
```

Comment:
I think it's beta? Or does this doc not distinguish?

Reply:
'Experimental' includes beta releases, so it's correct in being listed as experimental.